src/pytest/test_external.py

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) 2018-2020, Michael P. Howard
+# Copyright (c) 2021-2024, Auburn University
+# Part of azplugins, released under the BSD 3-Clause License.
+
+import hoomd
+from hoomd.azplugins import external
```

Comment:
I prefer not to use from imports in unittests because it lets us check that we have internally imported modules as we intend the user to be able to use them.
```suggestion
import hoomd.azplugins
```